data/World/Overworld.json

```diff
@@ -81,7 +81,25 @@
             "Showed Mido Sword & Shield": "is_child and Kokiri_Sword and Buy_Deku_Shield"
         },
         "locations": {
+            "KF Behind Midos Blue Rupee": "is_child",
+            "KF Boulder Maze Blue Rupee 1": "is_child",
+            "KF Boulder Maze Blue Rupee 2": "is_child",
+            "KF End of Bridge Blue Rupee": "is_child",
+            "KF Top of Sarias Recovery Heart 1": "is_child",
+            "KF Top of Sarias Recovery Heart 2": "is_child",
+            "KF Top of Sarias Recovery Heart 3": "is_child",
+            "KF Grass Near Ramp Green Rupee 1": "is_child",
+            "KF Grass Near Ramp Green Rupee 2": "is_child",
+            "KF Grass Near Midos Green Rupee 1": "is_child",
+            "KF Grass Near Midos Green Rupee 2": "is_child",
             "KF Kokiri Sword Chest": "is_child",
+            "KF Bean Platform Green Rupee 1": "is_adult and (can_plant_bean or can_use(Hover_Boots))",
+            "KF Bean Platform Green Rupee 2": "is_adult and (can_plant_bean or can_use(Hover_Boots))",
+            "KF Bean Platform Green Rupee 3": "is_adult and (can_plant_bean or can_use(Hover_Boots))",
+            "KF Bean Platform Green Rupee 4": "is_adult and (can_plant_bean or can_use(Hover_Boots))",
+            "KF Bean Platform Green Rupee 5": "is_adult and (can_plant_bean or can_use(Hover_Boots))",
+            "KF Bean Platform Green Rupee 6": "is_adult and (can_plant_bean or can_use(Hover_Boots))",
+            "KF Bean Platform Red Rupee": "is_adult and (can_plant_bean or can_use(Hover_Boots))",
```

Comment:
KF Bean Platform Rupees are currently only logically accessible using Hover Boots (unless beans are already planted, though that setting didn't exist when you created this) because the can_plant_bean helper checks for you currently being a child, since you need to be child to plant a bean, but the rupees are only accessible as an adult. You need here(can_plant_bean), so that child can open the way for adult. (You might consider a region for these rupees just to avoid the repetition of having to check these requirements every time?)

Reply:
Alright, you've resolved the bug. (The use of can_use around Hovers is not really necessary since all that does is check is_adult again, but some people like seeing it there regardless, so just a style choice really.) Any thoughts on whether or not to use a region?